src/FormBuilder.ts

```diff
@@ -205,6 +205,11 @@ interface IFormElement extends IEventHandler {
   value: any;
 
   /**
+   * Form element values for multiple selection
+   */
+  values?: {id: string, text: string}[];
```

Comment:
`IFormSelectElement` currently needs methods that FormSelect2 does not implement (e.g. updateOptionElements), which may not be needed
Maybe a new `IFormSelect2Element` with the `values` method would be better?